src/ontology/mondo-ingest.Makefile

```diff
@@ -351,14 +351,46 @@ deploy-mondo-ingest:
 
 USE_MONDO_RELEASE=false
 
+# TODO: make sure passes aftr add this line back below cd mondo/src/ontology
+#		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
+
 tmp/mondo_repo_built:
 	if [ $(USE_MONDO_RELEASE) = true ]; then wget http://purl.obolibrary.org/obo/mondo.owl -O $@; else cd $(TMPDIR) &&\
 		rm -rf ./mondo/ &&\
 		git clone --depth 1 https://github.com/monarch-initiative/mondo &&\
 		cd mondo/src/ontology &&\
 		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
+		latest_hash=$$(git rev-parse origin/master) &&\
 		cd ../../../../ &&\
-		touch $@; fi
+		echo "$$latest_hash" > $@; fi
+
+# TODO
+# explain what does here. explain that -B is just in case tmp/mondo_repo_built exists but not the rpeo
+.PHONY: refresh-mondo-clone
+refresh-mondo-clone:
+	@if [ ! -d tmp/mondo ]; then \
+		$(MAKE) tmp/mondo_repo_built -B; \
+	else \
+		current_hash=$$(cat tmp/mondo_repo_built); \
+		cd tmp/mondo; \
+		git fetch origin; \
+		latest_hash=$$(git rev-parse origin/master); \
+		if [ ! "$$current_hash" = "$$latest_hash" ]; then \
+			cd .. && mv mondo mondo-bak && mv mondo_repo_built mondo_repo_built-bak; \
+			cd .. && $(MAKE) tmp/mondo_repo_built -B; \
+			rm -rf tmp/mondo-bak tmp/mondo_repo_built-bak; \
+		fi; \
+	fi
+
+.PHONY: refresh-mondo-owl
+refresh-mondo-owl: refresh-mondo-clone
+
+.PHONY: refresh-mondo-sssom
+refresh-mondo-sssom: refresh-mondo-clone
```

Comment:
Note if you seperate the two, you might increase runtime, as the build may (may) run some of the same goals required for the mondo.owl and mondo.sssom.tsv multiple times.

Reply:
I'm wondering how this would increase how often `refresh-mondo-clone` would be called, but it shouldn't matter much, because this PR does two things:
1. Makes sure these files get updated whenever needed.
2. Makes sure they do not get recreated is they are up-to-date.

Regarding (2),as discussed in [the issue](https://github.com/monarch-initiative/mondo-ingest/issues/538), this goal will not do anything if the latest commit hash on `main` hasn't changed. So, if `refresh-mondo-clone` does get run more often than it should, the only downside here is dependency on a network connection, for the `git fetch` command. There is latency in running that command, but it is nominal.

Reply:
mondo.owl and mondo.sssom.tsv share some of the same targets in the Mondo (not mondo ingest!) make pipeline and they may be run twice. For example, both depend on the goal "filtered.obo". It is possible that this will add 5 minutes of runtime, but it is also possible no.

Reply:
Hmm, I guess it's not worth worrying about too much right now. I'll just say that I don't think that will be the case. I'm basically not changing how any of the goals in `mondo-ingest.Makefile` relate to `mondo.owl` and `mondo.sssom.tsv`, nor how those relate to `tmp/mondo/` / `tmp/mondo_repo_built`. Further, I kept `refresh-mondo-clone` (which is the new `tmp/mondo_repo_built` goal) the same as it was before, in that it runs `make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\`.

Therefore, whether `refresh-mondo-sssom` or `refresh-mondo-owl` runs doesn't matter; both of these files will be built together whenever one of them is called.

And, of course, subsequent calls of these goals will do nothing so long as no new commits have happened in `mondo`.